app/utils/config.example.js

```diff
@@ -0,0 +1,4 @@
+
+module.exports = {
+  GOOGLE_API_KEY : 'REPLACE_ME'
```

Comment:
If we're within the free usage tier's limits, no key is required: http://stackoverflow.com/questions/2769148/whats-the-api-key-for-in-google-maps-api-v3. Should we leave it out for now?